examples/companySurvey/main.bal

```diff
@@ -0,0 +1,21 @@
+import ballerina/io;
+import ballerinax/slack;
+
+public function companySurvey() returns error? {
+    slack:Client cl = check new ({
+        auth: {
+            token: value
```

Comment:
The token should be defined as configurable.

Reply:
Resolved Thanks